cli/standalone.rs

```diff
@@ -227,13 +232,24 @@ pub async fn run(
       .collect::<Vec<_>>(),
   );
 
+  let mut root_store = program_state
+    .root_store
+    .clone()
+    .or_else(|| Some(create_default_root_store()))
+    .unwrap();
+
+  if let Some(cert) = metadata.ca_data {
+    let reader = &mut BufReader::new(Cursor::new(cert));
+    root_store.add_pem_file(reader).unwrap();
+  }
+
```

Comment:
This section threw me for a loop for a bit but I think its accurate now. I had to understand how the ProgramState, Metadata and Flags all related and how its due to the standalone feature to get this.

The goal is to consolidate it all into a single RootCertStore but the RootCertStore is not serializable, thus the ca_stores and the ca_file are flags but when building standalone the ca_file gets serialized into ca_data, without the flag, so when launching the standalone it will create the base stores then add the stored cert from the ca_data.